src/libtrixi.h

```diff
@@ -0,0 +1,11 @@
+#ifndef LIBTRIXI_H_
+#define LIBTRIXI_H_
+
+#include <mpi.h> // required for MPI
+
+void trixi_initialize();
```

Comment:
```suggestion
void trixi_initialize(MPI_Fint* comm);
```
To ,act function definition?